meetings/2025-01-29.md

```diff
@@ -0,0 +1,87 @@
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/).
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+
+* elf Pavlik
+
+## Topics
+
+### [New Work Item] ActivityPub/Fediverse interoperability
+
+https://github.com/solid/specification/issues/708
+
+* eP: It is just an announcement, next week I hope to bring it up for a vote.
+* PAC: It is importantn to have people from Social Web CG also involved. We need consistent story about Solid and Activity Pub. 
+* HZ: Question to PAC, isn't it more important to bring it to LWS WG?
+* PAC: The charter mentions Social Web CG as a contact that we will synchronize with. There is no active Social Web WG. WGs have active scope, they need to sync, what we are discussin isn't directly in the scope of the LWS WG.
+* HZ: What is the best path forward to bring it to LWS
+* PAC: This is less urgent for the WG, at this moment it focuses on use cases. I don't expect strong coupling with Social Web CG. This work item should be a joined work in my opinion. Hopefully it will progress in CGs and we will have better understanding how to progress.
+* RG: The way we were discussin it yesterday, it will be layer on top of Solid, not something in the Solid spec. Everything should be workable as a layer on top of solid. 
+
+### State of CG reports
+
+https://www.w3.org/2024/09/linked-web-storage-wg-charter.html#dependencies
+
+> Depending on the Working Group progress, including consideration for adequate implementation experience, the Group may also decide to adopt the following dependencies as input documents
+
+* eP: It sounds like Solid-OIDC, ACP, WAC and Solid Notifications **MAY** be adopted by the WG. Where new implementation/deployment feedback should be addressed meanwhile?
+* eP: Possible inspiration for CG-WG coordination: https://github.com/w3c-fedid/Administration/blob/main/proposals-CG-WG.md
+* PAC: WG says that at this moment those reports are not formally considered as inputs from CGs. I would consider it ok to continue working on those reports in CG. WG may at some point signal that it wants to take over a specific report.
+* PAC: As for incubation in FedCM community group, I assume that this is browser centric and it may relate to how the browser vendors work. It might not transfer well in our community.
```

Comment:
```suggestion
* PAC: WG says that, at this moment, those reports are not formally considered as inputs from CGs. I would consider it OK to continue working on those reports in Solid CG. LWS WG may at some point signal that it wants to take over a specific report.
* PAC: As for incubation in FedCM community group, I assume that this is browser-centric and it may relate to how the browser vendors work. It might not transfer well to our community.
```